test/extensions/INTEL/SPV_INTEL_fpga_latency_control/IntelFPGALatencyControl.ll

```diff
@@ -0,0 +1,75 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
General comment:
Please rewrite the test using opaque pointers

Reply:
Done. Please check if its looks correct.